src/components/PrivateRouter.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+    Route,
+    Redirect
+} from "react-router-dom";
+
+import * as AuthSelectors from '../store/Auth/reducer'
+import {connect} from 'react-redux';
+
+
+const PrivateRoute = ({component: Component, isAuthenticated, ...rest}) => (
+    <Route
+        {...rest}
+        render={props =>
+            isAuthenticated ? (
+                <Component {...rest} {...props} />
+            ) : (
+                <Redirect
+                    to={{
+                        pathname: '/login',
+                        state: {from: props.location}
+                    }}
+                />
+            )
+        }
+    />
+);
+
+function MapStateToProps(state) {
+    console.log("isAuthenticated", AuthSelectors.isAuthenticated(state))
+    return {
```

Comment:
```javascript
return{
   store: state.AU_authState,
}
```

After insert this you should remove all imports Redux part! Since this is a separate level of abstraction, and it is better not to do extra work, because it is already doing redux

Reply:
Done